Tests/Tests/STPCardTest.m

```diff
@@ -151,38 +149,86 @@ - (void)testStringFromFunding {
 
 #pragma mark -
 
+- (void)testInitWithIDBrandLast4ExpMonthExpYearFunding {
+    STPCard *card = [[STPCard alloc] initWithID:@"card_1AVRojEOD54MuFwSxr93QJSx"
+                                          brand:STPCardBrandVisa
+                                          last4:@"5556"
+                                       expMonth:12
+                                        expYear:2034
+                                        funding:STPCardFundingTypeDebit];
+    XCTAssertEqualObjects(card.cardId, @"card_1AVRojEOD54MuFwSxr93QJSx");
+    XCTAssertEqual(card.brand, STPCardBrandVisa);
+    XCTAssertEqualObjects(card.last4, @"5556");
+    XCTAssertEqual(card.expMonth, (NSUInteger)12);
+    XCTAssertEqual(card.expYear, (NSUInteger)2034);
+    XCTAssertEqual(card.funding, STPCardFundingTypeDebit);
+}
+
+- (void)testInit {
+    STPCard *card = [[STPCard alloc] init];
+    XCTAssertEqual(card.brand, STPCardBrandUnknown);
+    XCTAssertEqual(card.funding, STPCardFundingTypeOther);
+}
+
 - (void)testLast4ReturnsCardNumberLast4WhenNotSet {
-    self.card.number = @"4242424242424242";
-    XCTAssertEqualObjects(self.card.last4, @"4242", @"last4 correctly returns the last 4 digits of the card number");
+    STPCard *card = [[STPCard alloc] init];
```

Comment:
can just do `[STPCard new]` instead of alloc/init? Or is that a style guide holy war? :)